pkg/virt-launcher/virtwrap/launchsecurity/sev.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021
+ *
+ */
+
+package launchsecurity
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+)
+
+const (
+	// Guest policy bits as defined in AMD SEV API specification
+	SEVPolicyNoDebug        uint = (1 << 0)
+	SEVPolicyEncryptedState uint = (1 << 2)
+)
+
+func SEVPolicyToBits(policy *v1.SEVPolicy) uint {
+	// NoDebug is always true
+	bits := uint(SEVPolicyNoDebug)
```

Comment:
`NoDebug` is always set to true just because of security reasons. If we do not explicitly disable debugging, the hypervisor will be able to decrypt and read the guest memory via the `DBG_DECRYPT/DBG_ENCRYPT` API. Something that we likely would prefer to avoid.

Reply:
I would say rather no, because its a big security hole that makes the whole idea of confidential VMs useless (potentially anyone with access to the hypervisor will be able to read sensitive data.... and that is what SEV primaraly tries to address).

Reply:
I tend to agree with @vasiliy-ul. We can always add such capability if someone needs it (and probably warn the user that of the risks involved). For now I think we can stick with the safer approach.

Thanks for the clarifications!